code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -224,13 +224,16 @@
 /datum/reagent/fuel/unholywater/on_mob_life(mob/living/M)
 	if(iscultist(M))
 		M.drowsyness = max(M.drowsyness-5, 0)
-		M.AdjustParalysis(-1, 0)
-		M.AdjustStunned(-2, 0)
-		M.AdjustWeakened(-2, 0)
-		M.adjustToxLoss(-2, 0)
-		M.adjustOxyLoss(-2, 0)
-		M.adjustBruteLoss(-2, 0)
-		M.adjustFireLoss(-2, 0)
+		M.adjustToxLoss(-3, 0)
+		M.adjustOxyLoss(-3, 0)
+		M.adjustBruteLoss(-3, 0)
+		M.adjustFireLoss(-3, 0)
+		M.adjustCloneLoss(-5, 0)
+		M.adjustBrainLoss(-3, 0)
+		if(iscarbon(M))
+			var/mob/living/carbon/C = M
+			if(C.blood_volume < 570) //the default blood amount is 560, this just gives people a nice little buffer. It is supposed to give you unholy vigour after all.
```

Comment:
Good change but you could probably use the define here: https://github.com/HippieStation/HippieStation13/blob/42915e2f0fc726df31f1cebd7f3206907862c391/code/__DEFINES/mobs.dm#L12